tests/stub/authorization/scripts/v4x3/reader_tx_yielding_error_on_commit.script

```diff
@@ -3,11 +3,24 @@
 A: HELLO {"{}": "*"}
 C: BEGIN {"mode": "r", "db": "adb"}
 S: SUCCESS {}
-C: RUN "RETURN 1 as n" {} {}
-   PULL {"n": 1000}
-S: SUCCESS {"fields": ["n"]}
-   RECORD [1]
-   SUCCESS {"type": "r"}
+{{
+    C: RUN "RETURN 1 as n" {} {}
+    S: SUCCESS {"fields": ["n"]}
+    C: PULL {"n": 1000}
+    S: RECORD [1]
+       SUCCESS {"type": "r"}
+----
+    C: RUN "RETURN 5 as n" {} {}
+    S: SUCCESS {"fields": ["n"]}
+    {{
+        C: PULL {"n": 1000}
+        S: RECORD [1]
+           SUCCESS {"type": "r"}
+    ----
+        C: DISCARD {"n": -1}
+        S: SUCCESS {}
+    }}
+}}
```

Comment:
How about this (similar for all versions of `reader_tx_yielding_error_on_commit.script`)?
```suggestion
C: RUN "RETURN 1 as n" {} {}
S: SUCCESS {"fields": ["n"]}
{{
    C: PULL {"n": {"Z": "*"}}
    S: RECORD [1]
----
    C: DISCARD {"n": {"Z": "*"}}
}}
S: SUCCESS {"type": "r"}
```

Reply:
If not this, it might be worth splitting the two scripts into separate files.